apps/web/src/views/Swap/V3Swap/containers/SwapCommitButton.tsx

```diff
@@ -361,7 +386,13 @@ export const SwapCommitButton = memo(function SwapCommitButton({
         }}
         id="swap-button"
         width="100%"
-        disabled={!isValid || (priceImpactSeverity > 3 && !isExpertMode) || !!swapCallbackError || !approved}
+        disabled={
+          !isValid ||
+          (priceImpactSeverity > 3 && !isExpertMode) ||
+          !!swapCallbackError ||
+          !approved ||
+          statusWallchain === 'pending'
+        }
```

Comment:
seems like disable status is toggling too frequently on bsc due to the revalidate time